botocore/client.py

```diff
@@ -77,6 +79,15 @@ def _load_service_model(self, service_name, api_version=None):
         self._register_retries(service_model)
         return service_model
 
+    def _get_client_class_name(self, service_model):
+        name = service_model.metadata.get(
+            'serviceAbbreviation',
+            service_model.metadata.get('serviceFullName'))
```

Comment:
Should we just fall back to `.service_name` if there's no serviceAbbreviation or serviceFullName?  It's is technically possible for this statement to evaluate to `None`.